navigator/frontend/src/applets/contract/ContractComponent.tsx

```diff
@@ -144,6 +144,11 @@ export default (props: Props) => {
         <p>{isArchived ? 'ARCHIVED' : null}</p>
         <ColumnContainer>
           <Column>
+              {contract.agreementText && contract.agreementText !== '' &&
+                (<span><SubHeader><Strong>Agreement Text</Strong></SubHeader>
```

Comment:
Consider using a component for the agreement text. An simple local function does the trick (although I suspect that you wrote a separate component and forgot to include it, see below):
``` 
const AgreementText = ({text}:{text: string}) => (
  <span>
    <SubHeader><Strong>Agreement Text</Strong></SubHeader>
    <span>{text}</span>
  </span>
);
```

and then 
```
{contract.agreementText && <AgreementText text={contract.agreementText} />}
```

Also note that `contract.agreementText` returns a falsy value for both `undefined` and an empty string.

Reply:
Thanks. Initially I had a separate component in a separate folder `AgreementText/[index|Guide].tsx`, but decided against it, as I wanted to use `SubHeader` without importing it in the other file. I'll move it to a separate function though.